README.md

```diff
@@ -1,49 +1,34 @@
 # macOS Cookbook
 
-[![Build Status](https://dev.azure.com/office/OE/_apis/build/status/Microsoft.macos-cookbook?branchName=master)](https://dev.azure.com/office/OE/_build/latest?definitionId=5072&branchName=master)
+[![Build Status](https://dev.azure.com/office/OE/_apis/build/status/microsoft.macos-cookbook?repoName=microsoft%2Fmacos-cookbook&branchName=master)](https://dev.azure.com/office/OE/_build/latest?definitionId=17314&repoName=microsoft%2Fmacos-cookbook&branchName=master)
 
 Chef resources and recipes for managing and provisioning macOS.
 
-- [Chef Requirements](#chef-requirements)
-- [Supported OS Versions](#supported-os-versions)
-- [Attributes](#attributes)
-- [Recipes](#recipes)
-- [Data Bags](#data-bags)
-- [Resources](#resources)
+- [Chef Requirements](#officially-supported-chef-versions)
+- [macOS Requirements](#officially-supported-os-versions)
+- [Included Resources](#resources)
 
 ## Officially Supported Chef Versions
 
 - Chef 16
 - Chef 17
 
-## Supported OS Versions
+## Officially Supported OS Versions
 
-- macOS High Sierra 10.13
-- macOS Mojave 10.14
-- macOS Catalina 10.15
-- macOS Big Sur 11.0
-
-## Attributes
-
-### Admin User and Password
-
-```ruby
-node['macos']['admin_user'] = 'vagrant'
-node['macos']['admin_password'] = 'vagrant'
-```
-
-Each of these attributes defaults to vagrant since our resources are developed
-with the Vagrant paradigm. In other words, the user and password declared here
-should be an admin user with passwordless super-user rights.
+- macOS 10.14 Mojave
+- macOS 10.15 Catalina
+- macOS 11 Big Sur
+- macos 12 Monterey
 
 ## Resources
+**The plist resource is deprecated and will be removed in the next major release of macos-cookbook. Please use the plist resource included with Chef Client >=16.**
```

Comment:
Do we know if the Chef Client version of the plist resource makes use of `defaults` in order to make sure the defaults cache is updated correctly?
I couldn't find any mention of `cfprefsd` or the `defaults` utility on Chef's page:
https://docs.chef.io/resources/plist/

Reply:
It does not, but neither does the macos `plist`. There are no plans to deprecate the `defaults` resource or any of the libraries it depends on.